changelog.md

```diff
@@ -6,11 +6,17 @@
 - bpk-scrim-utils:
   - Removed `padded` prop in favour of more generic `contentClassName`
   - Fixed modal white background on iPhone not showing
+- bpk-component-image:
+  - `withLazyLoading` now applys styling options to the wrapper div it creates rather than the lazy loaded component.
```

Comment:
*applies

Reply:
That's what I get for not reading what I write